### PR TITLE
Branch doc ppp

### PR DESCRIPTION
**Pull request by sopa301**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -705,6 +705,19 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+### Planned Enhancements
+1. The current `schedule` command does not check if the given date is before the current date, so it is vulnerable to
+user error. We plan to make the command check for the date and fail if the date is before the current date: `Date given
+ cannot be before the current date`.
+2. The current `phone` and `next-of-kin phone` fields currently only accepts numbers. It cannot accept international
+number formats. We plan to make the fields accept symbols so numbers such as `+6598765432` and `001-234-1-4610818`
+will be accepted. This will involve changing the validity checker for both fields.
+3. The current contact book does not check for duplicates beyond the exact matching of the person's `name`.
+We plan to refuse adding/editing of a person's details if it results in two people sharing a `name` (case-insensitive)
+or `phone` since two people are very unlikely to share those details.
```
Comment (jylow) [inline docs/DeveloperGuide.md:717]: People are likely to share name tho...

Comment (sopa301) [reply docs/DeveloperGuide.md:717]: I'm thinking maybe can put full name? We can decide this later anyway

File: docs/team/sopa301.md [hunk 2]
```diff
@@ -5,19 +5,40 @@ title: Poon Yip Hang, Ryan's Project Portfolio Page
...
-keywords for help command.
+* **Enhancements Implemented**:
+  * Added `next-of-kin` and `next-of-kin phone` fields and wrote tests
+    (Pull request [#43](https://github.com/AY2324S1-CS2103T-F12-1/tp/pull/43))
+  * Added confirm clear window for `clear` command
+    (Pull request [#75](https://github.com/AY2324S1-CS2103T-F12-1/tp/pull/75))
+  * Added available keywords for `help` command
+    (Pull request [#67](https://github.com/AY2324S1-CS2103T-F12-1/tp/pull/67))
+  * Enhanced `find` command to accept multiple names, tags and financial plans and wrote tests
+    (Pull request [#125](https://github.com/AY2324S1-CS2103T-F12-1/tp/pull/125))
```
Comment (Kb-Tay) [inline docs/team/sopa301.md:26]: Maybe can add justifications and highlights as shown in AB4.

**Review by Kb-Tay — APPROVED** [review]
LGTM. Maybe just need to standardise the implementation for person equality.

**Pull request merged**